cime_config/machines/Depends.ibmgpu.cmake

```diff
@@ -53,7 +53,7 @@ set(FILES_NEED_OPENMP_FLAGS
   eam/src/physics/crm/samomp/params.F90
   eam/src/physics/crm/samomp/dmdf.F90
   eam/src/physics/crm/samomp/domain.F90
-  eam/src/physics/crm/samomp/ecppvars.F90
+  eam/src/physics/crm/ecppvars.F90
```

Comment:
@whannah1 this path change appears to be done in E3SM-Project/E3SM#4301. Need to propagate it to other compilers too.

Reply:
Did you add this change here as a courtesy or does Blues have the IBM compiler?

Reply:
Just wanted to update all related Depends.* files in one PR. This will be needed for `ibmgpu` on Summit.